### DISCUSSION TITLE
Dealing with database changes when using dump of production database in development

### DISCUSSION THREAD

**Discussion by mkastner** [keystonejs/keystone#8806] [Questions]
I usually develop by using a dumped copy of the production database. For some reason which I have been unable to figure out I keep getting this message:

```
Drift detected: Your database schema is not in sync with your migration history.

The following is a summary of the differences between the expected database schem
a given your migrations files, and the actual schema of the database.

It should be understood as the set of changes to get from the expected schema to 
the actual schema.

[+] Added enums
  - DocumentPositionType
  
  
We need to reset the postgresql database "cmsfmh_development" at localhost:5432.
? Do you want to continue? All data will be lost. › (Y/n)
```

Of course I don't want to lose my data. I'd like to fix the inconsistencies.

Moreover when i deploy my application everything is fine. When I run keystone config changes i.e. migrations in production there's no complaint about differences which might require a reset of the database. So I would expect if there's no problem with the deployed database, then there shouldn't be a problem with a dumped copy either.

I have no clue I can fix this. There ought to be a way to derive or infer a keystone database config from an existing database structure.

**Comment by dcousens** [comment]
@mkastner can you reproduce this?
